src/authenticatedFetch/AutomaticRefreshFetcher.ts

```diff
@@ -60,6 +60,9 @@ export default class AutomaticRefreshHandler implements IAuthenticatedFetcher {
         );
       } catch (err) {
         // Do nothing
+        // Failure at this point probably indicates that the refresh token has
+        // been invalidated, and there is nothing that the client can do
+        // but authenticate again
```

Comment:
Always very scary to see exceptions being swallowed. Seems to beg the question - "But what if the developer wants to be notified that the refresh token has been invalidated?".
I'm also assuming that we have no logging capability in this code, as this would seem to warrant at least a `DEBUG(`Error trying to <whatever>. Details: {err}`);`

Reply:
There is no logging capabilities as part of the codebase. The `debug` module makes a lot of assumption about running in a Node environment (e.g. colored logs for the terminal, etc), so it's not very convenient to have it in a browser-oriented codebase. I could always throw in a `console.error` there, but I'm not sure that's good practice to write in the console from a low-level library

Reply:
Yeah, I don't know what 'best practice' is in JS-land for this kind of thing - maybe @Vinnl or @ajacksified have ideas, but fundamentally it just seems like swallowing any exception, ever, is just very, very **_bad_**, regardless of programming language...! (But at the very least, the code has a good comment to explain the possible use-cases that could result in this 'catch{}' block, but I wonder how (if?) we have 100% test coverage on this...?